src/test/mediapipeflow_test.cpp

```diff
@@ -140,6 +155,45 @@ TEST_F(MediapipeTFTest, DummyInfer) {
     checkDummyResponse("out", requestData, request, response, dummysInTheGraph, 1, modelName);
 }
 
+TEST_F(MediapipeTensorTest, DummyInfer) {
+    const ovms::Module* grpcModule = server.getModule(ovms::GRPC_SERVER_MODULE_NAME);
+    KFSInferenceServiceImpl& impl = dynamic_cast<const ovms::GRPCServerModule*>(grpcModule)->getKFSGrpcImpl();
+    ::KFSRequest request;
+    ::KFSResponse response;
+
+    const std::string modelName{"mpTensorDummy"};
+    request.Clear();
+    response.Clear();
+    inputs_info_t inputsMeta{{"in", {DUMMY_MODEL_SHAPE, precision}}};
+    std::vector<float> requestData{13.5, 0., 0, 0., 0., 0., 0., 0, 3., 67.};
+    preparePredictRequest(request, inputsMeta, requestData);
+    request.mutable_model_name()->assign(modelName);
+    ASSERT_EQ(impl.ModelInfer(nullptr, &request, &response).error_code(), grpc::StatusCode::OK);
+    size_t dummysInTheGraph = 1;
+    checkDummyResponse("out", requestData, request, response, dummysInTheGraph, 1, modelName);
+}
+
+TEST_F(MediapipeTfLiteTensorTest, DummyInfer) {
```

Comment:
We should also check scalar and 0-data scenario, for example method for creating 0-data tensor differs from creating one with data in OpenVINO

Reply:
TfLite Support is enabled only in calculator - not in OVMS